docs/pages/reference/config.mdx

```diff
@@ -610,9 +610,9 @@ proxy_service:
     # "v1": defaults to 0.0.0.0:3024
     tunnel_listen_addr: 0.0.0.0:3024
 
-    # The HTTPS listen address to serve the Web UI and also to authenticate the
-    # command line (CLI) users via password+HOTP
-    # Also handles the PostgreSQL proxy if database access is enabled.
+    # The HTTPS listen address to serve the Web UI and authenticate CLI (command
```

Comment:
Instead of "CLI (command line interface)", would it be clearer to say "tsh"?

Reply:
I think we can just say "authenticate users", I expect most of our clients go through this port (tsh, Connect, etc).